scrapy_poet/overrides.py

```diff
@@ -42,3 +47,67 @@ def overrides_for(self, request: Request) -> Mapping[Callable, Callable]:
         return self.get(get_domain(request.url), {})
 
 
+RuleAsTuple = Union[Tuple[str, Callable, Callable], List]
```

Comment:
Below, `RuleAsTuple` is solely used in code like: `Union[RuleAsTuple, OverrideRule]`.

It might be worth in defining it directly like this.

Reply:
Handled in https://github.com/scrapinghub/scrapy-poet/pull/56/commits/706e4ac6c5e3c1c533eeb66a81d413cc72adc518